lib/options.go

```diff
@@ -519,7 +520,7 @@ func (o Options) Apply(opts Options) Options {
 func (o Options) Validate() []error {
 	// TODO: validate all of the other options... that we should have already been validating...
 	// TODO: maybe integrate an external validation lib: https://github.com/avelino/awesome-go#validation
-	var errors []error
+	var errorsSlice []error
```

Comment:
nit: I'd suggest using something more explicit like `validationErrors`, rather than bringing the type into the variable name, which doesn't improve the readability of the code

Reply:
Thank you for the suggestion!
Fixed.